api/db-cdr/changelog-schema/db.changelog-18.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet author="danrodney" id="changelog-18">
+
+    <addColumn tableName="concept">
+      <!-- Contains the concept ID, concept code, concept name, and all concept_synonym_name values
```

Comment:
Test this concept synonym theory out  by populating them in old cdr20180920 one that is not used and see how it works. And test query speed . in mysql

Reply:
Also, what is the format of data going into this column? -- Ie, when we want to show synonyms how do we pull it out ?